### ISSUE TITLE
Update release script to install stable version of Admin Client

### ISSUE THREAD

**Issue by jonkoops** [keycloak/keycloak-ui#1034]
Once we are using pre-release versions of the Admin Client (as outlined in #1031) we will also have to update the release script so that a stable version of the Admin Client is used during the release process.

Specifically the [release script](https://github.com/keycloak/keycloak-admin-ui/blob/master/release.sh) will have to be updated so that it installs the stable version (`npm install @keycloak/keycloak-admin-client@$VERSION`). This needs to happen before the theme is built so the changes are included in the final result.

It is not clear if the environment that runs the build script actually has Node an NPM installled. It might be better to include this step as part of the Maven build (which will set up Node to build the application). If this is the case we only want to run this step on production builds somehow, perhaps through an environment variable or other CLI flag.

**Comment by ssilvert** [comment]
No longer relevant.
